schemas/answers/text_field.json

```diff
@@ -26,6 +26,9 @@
       "mandatory": {
         "type": "boolean"
       },
+      "open": {
```

Comment:
I personally think it is clearer for this to be `visible` or `is_visible`. Thoughts?

Reply:
visible - we already specify that it is a boolean so the `is_` seems superfluous.

Reply:
Ok, sorted now, thanks!

Reply:
The only thing is: "visible" is used in schemas sometimes(in obsolete "navigation"), but I discussed it with the guys and they gave me the green light.